tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_default_parameters.bal

```diff
@@ -190,6 +191,22 @@ function foo7(string s, float b = 1.1, FooObject o = new("default", 100)) return
     return [s, b, o];
 }
 
+function testFuncWithDefaultByteValue() {
+    byte b = funcWithDefaultByteValue();
```

Comment:
Shall we add test for call testFuncWithDefaultByteValue with some value and test that value.

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/31612/commits/44689930a2980827b8237aac68183c31e7421e0e